pkg/driver/nri_hooks.go

```diff
@@ -187,11 +217,6 @@ func (cp *CPUDriver) StopPodSandbox(_ context.Context, pod *api.PodSandbox) erro
 // RemovePodSandbox handles pod sandbox removal requests from the NRI.
 func (cp *CPUDriver) RemovePodSandbox(_ context.Context, pod *api.PodSandbox) error {
 	klog.Infof("RemovePodSandbox Pod %s/%s UID %s", pod.Namespace, pod.Name, pod.Uid)
-	// TODO(pravk03): If a pod with guaranteed CPUs is removed, we need to update the CPU assignment of other containers with shared CPUs.
-	// There isn't a way to do that in this call. It would only be updated the next time CreateContainer() gets called.
```

Comment:
the comment is probably still relevant, isn't it?

Reply:
Added it back.